src/partials/_carousel.hbs

```diff
@@ -47,6 +47,4 @@
 
 		</div>
 	</div>
-</section>
-
-<script src="/dist/index.js"></script>
```

Comment:
I believe this <script> tag should go in the <head> so that javascript will be loaded when the page loads. Similar to your CSS